README.md

```diff
@@ -278,36 +222,87 @@ Check out `tune --help` for all possible CLI commands and options. For more info
 
 &nbsp;
 
-## Design Principles
+---
+
+## Llama3 and Llama3.1
```

Comment:
why llama 3? I think its fine to just use 3.1. This may get stable soon, so may keeping it just as "llama" is better

Reply:
Yeah I will drop 3, probably will keep 3.1 in just to be explicit (otherwise it's not clear what we're referring to). One thing is that the tutorial in the docs is still for Llama3, not 3.1. So strictly speaking it's not 100% correct, but maybe that's a minor point